2024-09-02 08:31:38,670 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-02 08:31:38,670 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-02 08:31:38,671 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240902_083138-w0iqi9nd/logs/debug.log
2024-09-02 08:31:38,672 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240902_083138-w0iqi9nd/logs/debug-internal.log
2024-09-02 08:31:38,672 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-02 08:31:38,672 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-02 08:31:38,672 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: cm9xi9lr
2024-09-02 08:31:38,674 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/cm9xi9lr
2024-09-02 08:31:38,676 ERROR   MainThread:1467 [jupyter.py:save_history():436] Run pip install nbformat to save notebook history
2024-09-02 08:31:38,676 INFO    MainThread:1467 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-09-02 08:31:38,676 INFO    MainThread:1467 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-09-02 08:31:38,677 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-02 08:31:38,677 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-02 08:31:38,677 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-02 08:32:03,633 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4076] rendering history
2024-09-02 08:32:03,634 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4108] rendering summary
2024-09-02 08:32:03,648 INFO    MainThread:1467 [wandb_run.py:_footer_sync_info():4035] logging synced files
2024-09-02 08:32:04,611 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-02 08:32:04,612 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-02 08:32:04,617 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-02 08:32:04,618 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-02 08:32:04,622 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-02 08:32:04,622 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-02 08:32:04,623 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-02 08:32:04,666 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-02 08:32:05,536 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-02 08:32:05,906 INFO    MainThread:1467 [wandb_run.py:_on_init():2488] got version response upgrade_message: "wandb version 0.17.8 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-09-02 08:32:05,906 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-02 08:32:06,202 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-02 08:32:06,202 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-02 08:32:06,202 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-02 08:32:06,203 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-02 08:32:06,203 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-04 14:19:14,320 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-04 14:19:14,325 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-04 14:19:14,326 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-04 14:19:14,327 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_141914-k2tqgu9q/logs/debug.log
2024-09-04 14:19:14,327 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_141914-k2tqgu9q/logs/debug-internal.log
2024-09-04 14:19:14,328 INFO    MainThread:1467 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x12b083b90>
2024-09-04 14:19:14,329 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-04 14:19:14,330 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-04 14:19:14,331 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: w0iqi9nd
2024-09-04 14:19:14,349 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/w0iqi9nd
2024-09-04 14:19:14,360 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-04 14:19:14,362 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-04 14:19:14,383 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-04 14:23:48,936 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 642, in init
    latest_run.finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 450, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2147, in finish
    return self._finish(exit_code, quiet)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2181, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2430, in _atexit_cleanup
    self._on_finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2695, in _on_finish
    _ = exit_handle.wait(timeout=-1, on_progress=self._on_progress_exit)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-04 15:11:46,934 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-04 15:11:46,948 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-04 15:11:46,949 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-04 15:11:46,949 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-04 15:11:46,949 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-04 15:11:46,949 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_151146-1c8w3i0n/logs/debug.log
2024-09-04 15:11:46,950 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_151146-1c8w3i0n/logs/debug-internal.log
2024-09-04 15:11:46,950 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-04 15:11:46,950 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-04 15:11:46,950 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-04 15:11:46,951 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-04 15:11:46,954 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-04 15:11:46,957 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-04 15:11:46,961 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-04 15:11:46,961 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-04 15:11:46,962 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-04 15:11:46,992 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-04 15:11:47,633 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-04 15:13:17,248 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-04 15:13:17,251 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-04 15:14:25,598 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-04 15:14:25,609 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-04 15:14:25,609 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-04 15:14:25,609 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-04 15:14:25,609 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-04 15:14:25,609 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_151425-gu8gch04/logs/debug.log
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_151425-gu8gch04/logs/debug-internal.log
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-04 15:14:25,610 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-04 15:14:25,611 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-04 15:14:25,612 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-04 15:14:25,614 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-04 15:14:25,616 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-04 15:14:25,620 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-04 15:14:25,620 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-04 15:14:25,620 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-04 15:14:25,654 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-04 15:15:55,913 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-04 15:15:55,916 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-04 16:53:05,638 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-04 16:53:05,657 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-04 16:53:05,658 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-04 16:53:05,659 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_165305-itniujel/logs/debug.log
2024-09-04 16:53:05,659 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_165305-itniujel/logs/debug-internal.log
2024-09-04 16:53:05,659 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-04 16:53:05,659 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-04 16:53:05,660 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-04 16:53:05,660 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-04 16:53:05,667 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-04 16:53:05,668 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-04 16:53:05,879 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-04 16:53:05,880 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-04 16:53:05,880 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-04 16:53:05,925 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-04 16:53:06,747 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-04 16:53:36,868 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-04 16:53:37,031 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-04 16:54:07,432 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-04 16:54:07,433 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-04 16:54:07,434 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-04 16:54:07,435 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-04 16:54:07,440 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-04 16:58:32,014 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-04 16:58:32,016 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-04 16:58:32,017 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-04 16:58:32,017 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-04 16:58:32,017 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-04 16:58:32,017 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-04 16:58:32,017 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-04 16:58:32,018 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-04 16:58:32,018 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-04 16:58:32,019 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_165831-80skpfw4/logs/debug.log
2024-09-04 16:58:32,020 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240904_165831-80skpfw4/logs/debug-internal.log
2024-09-04 16:58:32,020 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-04 16:58:32,021 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-04 16:58:32,021 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: itniujel
2024-09-04 16:58:32,027 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/itniujel
2024-09-04 16:58:32,028 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-04 16:58:32,030 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-04 16:58:32,032 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-04 16:58:32,219 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-04 16:59:07,013 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 642, in init
    latest_run.finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 450, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2147, in finish
    return self._finish(exit_code, quiet)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2181, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2430, in _atexit_cleanup
    self._on_finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2687, in _on_finish
    result = server_info_handle.wait(timeout=-1)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 10:24:27,045 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 10:24:27,075 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 10:24:27,075 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 10:24:27,075 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 10:24:27,076 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 10:24:27,076 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 10:24:27,076 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 10:24:27,076 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 10:24:27,076 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 10:24:27,077 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 10:24:27,077 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_102427-i5gin8zn/logs/debug.log
2024-09-05 10:24:27,077 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_102427-i5gin8zn/logs/debug-internal.log
2024-09-05 10:24:27,078 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 10:24:27,078 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 10:24:27,079 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 10:24:27,080 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 10:24:27,091 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 10:24:27,093 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 10:24:27,130 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 10:24:27,131 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 10:24:27,131 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 10:24:27,210 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 10:24:28,203 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 10:24:58,288 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-05 10:24:58,386 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 10:24:58,721 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-05 10:24:58,722 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-05 10:24:58,723 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-05 10:24:58,723 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-05 10:24:58,724 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-05 11:01:12,931 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:01:12,934 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:01:12,934 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:01:12,934 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:01:12,934 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:01:12,934 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:01:12,935 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:01:12,935 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:01:12,935 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:01:12,935 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_110112-l7006e4t/logs/debug.log
2024-09-05 11:01:12,935 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_110112-l7006e4t/logs/debug-internal.log
2024-09-05 11:01:12,936 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:01:12,936 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:01:12,936 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: i5gin8zn
2024-09-05 11:01:12,939 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/i5gin8zn
2024-09-05 11:01:12,941 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-05 11:01:12,943 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-05 11:01:12,945 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-05 11:01:41,240 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4076] rendering history
2024-09-05 11:01:41,246 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4108] rendering summary
2024-09-05 11:01:41,268 INFO    MainThread:1467 [wandb_run.py:_footer_sync_info():4035] logging synced files
2024-09-05 11:01:43,334 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:01:43,340 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:01:43,344 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:01:43,346 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:01:43,361 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:01:43,365 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:01:43,398 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:01:43,464 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:01:44,440 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 11:01:46,564 INFO    MainThread:1467 [wandb_run.py:_on_init():2488] got version response upgrade_message: "wandb version 0.17.8 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-09-05 11:01:46,568 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-05 11:01:46,913 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-05 11:01:46,916 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-05 11:01:46,916 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-05 11:01:46,917 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-05 11:01:46,918 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-05 11:03:02,743 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:03:02,749 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:03:02,749 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:03:02,749 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:03:02,750 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_110302-xejkvird/logs/debug.log
2024-09-05 11:03:02,768 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_110302-xejkvird/logs/debug-internal.log
2024-09-05 11:03:02,768 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:03:02,769 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:03:02,769 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: l7006e4t
2024-09-05 11:03:02,771 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/l7006e4t
2024-09-05 11:03:02,772 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-05 11:03:02,773 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-05 11:03:02,774 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-05 11:03:07,656 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4076] rendering history
2024-09-05 11:03:07,657 INFO    MainThread:1467 [wandb_run.py:_footer_history_summary_info():4108] rendering summary
2024-09-05 11:03:07,670 INFO    MainThread:1467 [wandb_run.py:_footer_sync_info():4035] logging synced files
2024-09-05 11:03:08,001 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:03:08,002 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:03:08,004 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:03:08,005 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:03:08,007 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:03:08,008 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:03:08,008 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:03:08,036 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:03:09,575 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 11:03:09,993 INFO    MainThread:1467 [wandb_run.py:_on_init():2488] got version response upgrade_message: "wandb version 0.17.8 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-09-05 11:03:09,993 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-05 11:03:10,329 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-05 11:03:10,330 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-05 11:03:10,330 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-05 11:03:10,331 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-05 11:03:10,332 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-05 11:18:16,601 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:18:16,604 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:18:16,605 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:18:16,605 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_111816-abdbds0i/logs/debug.log
2024-09-05 11:18:16,605 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_111816-abdbds0i/logs/debug-internal.log
2024-09-05 11:18:16,606 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:18:16,606 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:18:16,606 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: xejkvird
2024-09-05 11:18:16,608 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/xejkvird
2024-09-05 11:18:16,609 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-05 11:18:16,610 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-05 11:18:16,610 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-05 11:18:23,925 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 642, in init
    latest_run.finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 450, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2147, in finish
    return self._finish(exit_code, quiet)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2181, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2430, in _atexit_cleanup
    self._on_finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2695, in _on_finish
    _ = exit_handle.wait(timeout=-1, on_progress=self._on_progress_exit)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:19:28,623 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:19:28,643 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:19:28,644 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:19:28,644 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:19:28,649 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:19:28,649 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:19:28,650 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:19:28,650 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:19:28,650 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:19:28,651 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:19:28,651 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_111928-wqngty05/logs/debug.log
2024-09-05 11:19:28,652 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_111928-wqngty05/logs/debug-internal.log
2024-09-05 11:19:28,652 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:19:28,653 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:19:28,653 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:19:28,653 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:19:28,656 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:19:28,656 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:19:28,664 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:19:28,664 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:19:28,665 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:19:28,693 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:20:59,138 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 11:20:59,147 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 11:21:07,665 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:21:07,676 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:21:07,677 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:21:07,678 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:21:07,678 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:21:07,678 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:21:07,678 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:21:07,679 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:21:07,680 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:21:07,681 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:21:07,681 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112107-nso055ga/logs/debug.log
2024-09-05 11:21:07,681 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112107-nso055ga/logs/debug-internal.log
2024-09-05 11:21:07,681 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:21:07,682 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:21:07,682 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:21:07,683 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:21:07,684 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:21:07,685 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:21:07,691 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:21:07,691 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:21:07,692 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:21:07,716 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:21:08,798 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 11:21:35,009 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 800, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:21:56,313 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:21:56,343 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:21:56,344 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:21:56,344 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:21:56,344 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:21:56,344 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:21:56,345 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:21:56,345 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:21:56,345 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:21:56,345 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:21:56,345 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112156-0f4vgynw/logs/debug.log
2024-09-05 11:21:56,346 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112156-0f4vgynw/logs/debug-internal.log
2024-09-05 11:21:56,346 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:21:56,346 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:21:56,346 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:21:56,346 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:21:56,348 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:21:56,348 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:21:56,351 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:21:56,352 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:21:56,352 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:21:56,369 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:23:26,697 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 11:23:26,707 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 11:25:25,911 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:25:25,928 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:25:25,929 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:25:25,929 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:25:25,929 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:25:25,929 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:25:25,930 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:25:25,930 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:25:25,930 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:25:25,930 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:25:25,931 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112525-rc295fk1/logs/debug.log
2024-09-05 11:25:25,932 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112525-rc295fk1/logs/debug-internal.log
2024-09-05 11:25:25,932 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:25:25,933 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:25:25,933 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:25:25,934 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:25:25,937 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:25:25,939 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:25:25,946 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:25:25,946 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:25:25,947 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:25:25,971 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:25:27,401 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 800, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:25:27,431 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 11:25:52,055 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:25:52,070 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:25:52,071 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:25:52,072 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:25:52,072 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:25:52,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:25:52,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:25:52,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:25:52,074 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:25:52,074 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:25:52,074 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112552-cqmfti8o/logs/debug.log
2024-09-05 11:25:52,075 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_112552-cqmfti8o/logs/debug-internal.log
2024-09-05 11:25:52,075 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:25:52,076 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:25:52,076 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:25:52,077 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:25:52,080 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:25:52,082 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:25:52,088 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:25:52,089 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:25:52,090 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:25:52,128 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:26:05,983 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 800, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:33:47,882 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:33:47,921 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:33:47,921 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:33:47,922 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:33:47,922 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:33:47,923 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:33:47,924 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:33:47,925 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:33:47,926 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:33:47,927 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:33:47,928 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113347-k82yxn7l/logs/debug.log
2024-09-05 11:33:47,929 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113347-k82yxn7l/logs/debug-internal.log
2024-09-05 11:33:47,929 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:33:47,930 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:33:47,930 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:33:47,930 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:33:47,933 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:33:47,934 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:33:47,953 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:33:47,953 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:33:47,953 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:33:47,985 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:33:49,485 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 11:33:58,208 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 800, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:34:13,998 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:34:14,019 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:34:14,020 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:34:14,020 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:34:14,021 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:34:14,021 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:34:14,022 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:34:14,023 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:34:14,023 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:34:14,024 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:34:14,026 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113414-4aqr1sq1/logs/debug.log
2024-09-05 11:34:14,027 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113414-4aqr1sq1/logs/debug-internal.log
2024-09-05 11:34:14,027 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:34:14,028 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:34:14,028 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:34:14,028 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:34:14,030 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:34:14,032 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:34:14,036 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:34:14,036 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:34:14,036 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:34:14,135 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:34:15,256 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 11:34:42,928 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 846, in init
    run._on_init()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2483, in _on_init
    version_result = version_handle.wait(timeout=30)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 11:35:00,263 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 11:35:00,316 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:35:00,318 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:35:00,319 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:35:00,319 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:35:00,320 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:35:00,320 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:35:00,320 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:35:00,321 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:35:00,321 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:35:00,321 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113500-vy5ooasb/logs/debug.log
2024-09-05 11:35:00,322 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_113500-vy5ooasb/logs/debug-internal.log
2024-09-05 11:35:00,322 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:35:00,323 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:35:00,323 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 11:35:00,324 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 11:35:00,328 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 11:35:00,329 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 11:35:00,333 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 11:35:00,334 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 11:35:00,334 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 11:35:00,415 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 11:35:04,469 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 11:35:05,016 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 11:35:34,550 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-05 11:35:34,640 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 11:35:34,885 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-05 11:35:34,887 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-05 11:35:34,887 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-05 11:35:34,888 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-05 11:35:34,890 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-05 11:54:52,525 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 11:54:52,535 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 11:54:52,535 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 11:54:52,536 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 11:54:52,536 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 11:54:52,536 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 11:54:52,536 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 11:54:52,537 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 11:54:52,537 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 11:54:52,537 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_115452-rkba511s/logs/debug.log
2024-09-05 11:54:52,538 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_115452-rkba511s/logs/debug-internal.log
2024-09-05 11:54:52,539 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 11:54:52,539 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 11:54:52,539 INFO    MainThread:1467 [wandb_init.py:init():632] re-initializing run, found existing run on stack: vy5ooasb
2024-09-05 11:54:52,541 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/vy5ooasb
2024-09-05 11:54:52,543 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-05 11:54:52,544 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-05 11:54:52,545 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-05 11:55:14,370 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 12:00:05,745 WARNING MainThread:1467 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 642, in init
    latest_run.finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 450, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2147, in finish
    return self._finish(exit_code, quiet)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2181, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2430, in _atexit_cleanup
    self._on_finish()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2687, in _on_finish
    result = server_info_handle.wait(timeout=-1)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-09-05 12:00:13,055 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 12:00:13,070 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 12:00:13,071 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 12:00:13,072 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 12:00:13,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 12:00:13,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 12:00:13,073 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 12:00:13,087 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 12:00:13,088 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 12:00:13,088 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 12:00:13,089 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_120013-rgn0s3n3/logs/debug.log
2024-09-05 12:00:13,090 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_120013-rgn0s3n3/logs/debug-internal.log
2024-09-05 12:00:13,091 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 12:00:13,092 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 12:00:13,092 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 12:00:13,092 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 12:00:13,095 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 12:00:13,096 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 12:00:13,113 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 12:00:13,117 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 12:00:13,119 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 12:00:13,173 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 12:00:15,690 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 12:01:43,439 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 12:01:43,440 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 12:02:23,548 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 12:02:23,566 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 12:02:23,568 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 12:02:23,569 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 12:02:23,569 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 12:02:23,569 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 12:02:23,570 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 12:02:23,573 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 12:02:23,575 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 12:02:23,576 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 12:02:23,576 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_120223-e7j93kmj/logs/debug.log
2024-09-05 12:02:23,576 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_120223-e7j93kmj/logs/debug-internal.log
2024-09-05 12:02:23,577 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 12:02:23,578 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 12:02:23,579 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 12:02:23,580 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 12:02:23,585 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 12:02:23,586 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 12:02:23,592 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 12:02:23,593 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 12:02:23,594 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 12:02:23,665 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 12:02:24,756 INFO    MainThread:1467 [wandb_run.py:_on_init():2479] communicating current version
2024-09-05 12:02:54,890 INFO    MainThread:1467 [wandb_init.py:init():848] starting run threads in backend
2024-09-05 12:03:25,245 INFO    MainThread:1467 [wandb_run.py:_console_start():2457] atexit reg
2024-09-05 12:03:25,247 INFO    MainThread:1467 [wandb_run.py:_redirect():2303] redirect: wrap_raw
2024-09-05 12:03:25,247 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-05 12:03:25,249 INFO    MainThread:1467 [wandb_run.py:_redirect():2368] Wrapping output streams.
2024-09-05 12:03:25,249 INFO    MainThread:1467 [wandb_run.py:_redirect():2393] Redirects installed.
2024-09-05 12:03:25,251 INFO    MainThread:1467 [wandb_init.py:init():891] run started, returning control to user process
2024-09-05 15:33:10,258 INFO    MainThread:1467 [wandb_run.py:_finish():2154] finishing run 48242293-colegio-wolfsohn-tabacinic/model-comparison/e7j93kmj
2024-09-05 15:33:10,304 INFO    MainThread:1467 [wandb_run.py:_atexit_cleanup():2418] got exitcode: 0
2024-09-05 15:33:10,306 INFO    MainThread:1467 [wandb_run.py:_restore():2400] restore
2024-09-05 15:33:10,307 INFO    MainThread:1467 [wandb_run.py:_restore():2406] restore done
2024-09-05 15:35:40,195 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-05 15:35:40,217 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-05 15:35:40,218 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-05 15:35:40,219 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-05 15:35:40,220 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-05 15:35:40,221 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-05 15:35:40,222 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-05 15:35:40,223 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-05 15:35:40,223 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-05 15:35:40,224 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-05 15:35:40,227 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_153540-5f038qds/logs/debug.log
2024-09-05 15:35:40,228 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240905_153540-5f038qds/logs/debug-internal.log
2024-09-05 15:35:40,229 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-05 15:35:40,230 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-05 15:35:40,230 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-05 15:35:40,231 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-05 15:35:40,236 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-05 15:35:40,248 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-05 15:35:40,289 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-05 15:35:40,290 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-05 15:35:40,291 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-05 15:35:40,324 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-05 15:37:10,676 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-05 15:37:10,678 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:47:21,231 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-09 07:47:21,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-09 07:47:21,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-09 07:47:21,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-09 07:47:21,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-09 07:47:21,265 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-09 07:47:21,267 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-09 07:47:21,268 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-09 07:47:21,269 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-09 07:47:21,269 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-09 07:47:21,270 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_074721-rsj8wcym/logs/debug.log
2024-09-09 07:47:21,271 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_074721-rsj8wcym/logs/debug-internal.log
2024-09-09 07:47:21,272 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-09 07:47:21,273 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-09 07:47:21,273 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-09 07:47:21,274 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-09 07:47:21,281 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-09 07:47:21,286 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-09 07:47:21,318 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-09 07:47:21,319 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-09 07:47:21,320 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-09 07:47:21,368 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-09 07:47:22,361 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-09 07:47:23,270 WARNING MsgRouterThr:1467 [router.py:message_loop():77] message_loop has been closed
2024-09-09 07:48:51,719 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:48:51,720 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:51:58,215 INFO    MainThread:1467 [wandb_settings.py:_apply_login():1936] Applying login settings: {}
2024-09-09 07:51:58,256 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2024-09-09 07:51:58,257 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Configure stats pid to 1467
2024-09-09 07:51:58,258 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/.config/wandb/settings
2024-09-09 07:51:58,258 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/settings
2024-09-09 07:51:58,259 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-09 07:51:58,260 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-09-09 07:51:58,261 INFO    MainThread:1467 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-09-09 07:51:58,265 INFO    MainThread:1467 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_075158-3xel5nak/logs/debug.log
2024-09-09 07:51:58,267 INFO    MainThread:1467 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gabrielslotnisky/Documents/GitHub/Proyecto2024/wandb/run-20240909_075158-3xel5nak/logs/debug-internal.log
2024-09-09 07:51:58,267 INFO    MainThread:1467 [wandb_init.py:init():607] calling init triggers
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():657] starting backend
2024-09-09 07:51:58,269 INFO    MainThread:1467 [wandb_init.py:init():661] setting up manager
2024-09-09 07:51:58,271 INFO    MainThread:1467 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-09-09 07:51:58,272 INFO    MainThread:1467 [wandb_init.py:init():669] backend started and connected
2024-09-09 07:51:58,276 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1338] probe notebook
2024-09-09 07:51:58,277 INFO    MainThread:1467 [wandb_run.py:_label_probe_notebook():1348] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-09 07:51:58,277 INFO    MainThread:1467 [wandb_init.py:init():764] updated telemetry
2024-09-09 07:51:58,327 INFO    MainThread:1467 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2024-09-09 07:53:28,701 ERROR   MainThread:1467 [wandb_init.py:init():823] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-09-09 07:53:28,711 ERROR   MainThread:1467 [wandb_init.py:init():1235] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 829, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
